include/analyzedphenotypes.admin.form.inc

```diff
@@ -1062,409 +1062,274 @@ function analyzedphenotypes_dragdrop($form, &$form_state, $source_name) {
-        }
-        else {
-          $varprop = analyzedphenotypes_cvprop('get_cvterm', array(
-            'cvterm_id' => $var_value,
-          ));
-
-          $default_var_cv = $varprop['cv_id'];
-        }
-
-        $form['ap_cv_select_field' . $k] = array(
-          '#type' => 'select',
-          '#title' => t('Locate ' . $title . ' cvterm: Controlled Vocabulary'),
-          '#required' => TRUE,
-          '#options' => $cv_options,
-          '#default_value' => $default_var_cv,
-          //
-          '#ajax' => array(
-            'event' => 'change',
-            'callback' => 'analyzedphenotypes_AJAX_callback',
-            'case' => 'varcvterm' . $title,
-            'wrapper' => 'ap-AJAX-wrapper-varcvterm' . $title,
-            'progress' => array('type' => 'throbber', 'message' => 'Loading cvterms...'),
-          ),
-          '#id' => 'ap-cv-select-field' . $k,
-        );
-
-        $cvvar_field_value = (isset($form_state['values']['ap_cv_select_field' . $k]))
-          ? $form_state['values']['ap_cv_select_field' . $k] : 0;
-
-        if ($cvvar_field_value > 0) {
-          $default_var_cv = $cvvar_field_value;
-        }
-
-        if ($default_var_cv > 0) {
-          $default_cvterm = $var_value;
-          $cvterms = analyzedphenotypes_cvprop('terms_in_cv', array(
-            'cv_id' => $default_var_cv,
-          ));
+        // Find if a config has been set.
 
-          $cvterm_options = array_map(function($t) {
-            return $t['name'];
-          }, $cvterms);
-        }
-        else {
-          $default_cvterm = 0;
-          $cvterm_options = array();
+        $var_value = variable_get($basename . $genus_key . '_' . $var_token[$j][0]);
+        if (isset($var_value)) {
+          $field_default = $var_value;
         }
 
-        // AJAX callback wrapper.
-        $form['ap_AJAX_wrapper_varcvterm' . $title] = array(
-          '#prefix' => '<div id="ap-AJAX-wrapper-varcvterm' . $title . '">',
-          '#suffix' => '</div>',
-        );
-
-        $form['ap_AJAX_wrapper_varcvterm' . $title][$var] = array(
+        $field_prop = array(
           '#type' => 'select',
           '#title' => '',
-          '#options' => $cvterm_options,
-          '#default_value' => $default_cvterm,
-          '#description' => t('Select ' . $title . ' cvterm.'),
-          '#id' => $term,
+          '#options' => array(0 => $var_token[$j][1]) + $field_options,
+          '#attributes' => array('class' => array('ap-table-select-field')),
+          '#default_value' => $field_default,
         );
+
+        $field_name = $basename . $genus_key . '_' . $var_token[$j][0];
       }
 
-      // Allow new traits?
-      $var_allow_new = $vars['ap_allow_new'];
-      $var_allow_new_value = variable_get($var_allow_new);
-      $default_allow_new = ($var_allow_new_value == 'not set') ? 0 : $var_allow_new_value;
+      $fields[ 'field_row_' . $i ][$field_name] = $field_prop;
```

Comment:
Inconsistency: on line 1124 you set the row to be `'ap_field_row_' . $i` but now here you assign the row to `'field_row_' . $i`